.github/jobs/ducks-backend.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+ENVIRONMENT=${ENVIRONMENT:-"develop"}
+
+BUCKET_NAME="web-ducks-server-$ENVIRONMENT"
```

Comment:
change the name to something that indicate that is a bucket for terrraform state

something like tf-state-ducks...

Reply:
ok